<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>学生管理系统</string>
  </property>
  <property name="windowFlags">
   <set>Qt::FramelessWindowHint</set>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f0f2f5;
    border-radius: 10px;
}

#titleBar {
    background-color: #ffffff;
    border-top-left-radius: 10px;
    border-top-right-radius: 10px;
    border: none;
}

#titleLabel {
    color: #333333;
    font-size: 16px;
    font-weight: bold;
}

#minimizeButton, #maximizeButton, #closeButton {
    border: none;
    border-radius: 15px;
    min-width: 30px;
    max-width: 30px;
    min-height: 30px;
    max-height: 30px;
    padding: 0px;
    margin: 0px 5px;
}

#minimizeButton {
    background-color: #ffbd44;
}

#maximizeButton {
    background-color: #00ca4e;
}

#closeButton {
    background-color: #ff605c;
}

#minimizeButton:hover, #maximizeButton:hover, #closeButton:hover {
    background-color: #404040;
}

QFrame#left_panel {
    background-color: #ffffff;
    border-radius: 10px;
    margin: 5px;
    border: none;
    box-shadow: 0 2px 8px rgba(0, 0, 0, 0.1);
}

QPushButton {
    background-color: #1890ff;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 8px 15px;
    font-size: 14px;
    box-shadow: 0 2px 4px rgba(24, 144, 255, 0.2);
}

QPushButton:hover {
    background-color: #40a9ff;
    box-shadow: 0 4px 8px rgba(24, 144, 255, 0.3);
}

QPushButton:pressed {
    background-color: #096dd9;
    box-shadow: 0 2px 4px rgba(24, 144, 255, 0.2);
}

QLabel {
    color: #333333;
    font-weight: bold;
    padding: 5px;
    border: none;
}

QLineEdit {
    border: 1px solid #e8e8e8;
    border-radius: 4px;
    padding: 5px;
    background-color: white;
    box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.05);
}

QLineEdit:focus {
    border: 1px solid #40a9ff;
    box-shadow: 0 0 0 2px rgba(24, 144, 255, 0.2);
}

QTableWidget {
    background-color: white;
    border: none;
    border-radius: 10px;
    gridline-color: #e8e8e8;
    box-shadow: 0 2px 8px rgba(0, 0, 0, 0.1);
}

QTableWidget::item {
    padding: 5px;
    border: none;
}

QTableWidget::item:selected {
    background-color: #e6f7ff;
    color: #1890ff;
    border: none;
}

QHeaderView::section {
    background-color: #fafafa;
    padding: 5px;
    border: none;
    border-bottom: 1px solid #f0f0f0;
    font-weight: bold;
}

QScrollBar:vertical {
    border: none;
    background-color: #f0f0f0;
    width: 10px;
    margin: 0px;
}

QScrollBar::handle:vertical {
    background-color: #d9d9d9;
    border-radius: 5px;
    min-height: 20px;
}

QScrollBar::handle:vertical:hover {
    background-color: #bfbfbf;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    border: none;
    background: none;
}

QScrollBar::add-page:vertical,
QScrollBar::sub-page:vertical {
    background: none;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="titleBar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>学生管理系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="minimizeButton">
         <property name="text">
          <string>—</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="maximizeButton">
         <property name="text">
          <string>□</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>×</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="contentWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QFrame" name="left_panel">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>141</width>
                <height>41</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Microsoft YaHei UI</family>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>操作面板</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="add_button">
              <property name="minimumSize">
               <size>
                <width>141</width>
                <height>51</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>添加学生</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="edit_button">
              <property name="minimumSize">
               <size>
                <width>141</width>
                <height>51</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>编辑学生</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="delete_button">
              <property name="minimumSize">
               <size>
                <width>141</width>
                <height>51</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: #ff4d4f;
}
QPushButton:hover {
    background-color: #ff7875;
}
QPushButton:pressed {
    background-color: #d9363e;
}</string>
              </property>
              <property name="text">
               <string>删除学生</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>141</width>
                <height>41</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Microsoft YaHei UI</family>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>搜索</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="search_input">
              <property name="minimumSize">
               <size>
                <width>141</width>
                <height>31</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>搜索学生...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="search_button">
              <property name="minimumSize">
               <size>
                <width>141</width>
                <height>51</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: #52c41a;
}
QPushButton:hover {
    background-color: #73d13d;
}
QPushButton:pressed {
    background-color: #389e0d;
}</string>
              </property>
              <property name="text">
               <string>搜索</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QTableWidget" name="table">
            <property name="minimumSize">
             <size>
              <width>611</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
